sqlrunner/__main__.py

```diff
@@ -71,6 +71,11 @@ def parse_args(args, environ):
         "--include-statistics",
         action="store_true",
     )
+    parser.add_argument(
+        "--generate-column-headers",
+        action="store_true",
+        help="Generate column headers from the query in the input SQL file and write to dummy data file",
+    )
```

Comment:
I don't think the desired behaviour should be turned on (or off) by the CLI. I'll explain in an overall comment.